### ISSUE TITLE
Outgoing Server (SMTP) settings disappearing

### ISSUE THREAD

**Issue by pglpm** [Betterbird/thunderbird-patches#42]
Betterbird 91.7.0-bb28 (64-bit) on Windows 10.

I opened the Account Settings and then the Outgoing Server (SMTP) settings, and saw that some (not all) SMTP servers I had added were gone. Most of them were associated with extra identities of the main account.

I tried to send an email from two of those identities, and was requested the SMTP password (which had been saved before). So it seems that the disappeared SMTP settings were still saved somewhere, except the password. Indeed, after sending those emails and closing and opening again the Outgoing Server settings, those SMTP identities reappeared – but not the others.

Let me know if  I can share any other info to help with this. Cheers!

**Comment by pglpm** [comment]
I'm using Betterbird, not Thunderbird, so I don't see why I should report this as a Thunderbird issue – then I can just as well use Thunderbird instead of Betterbird.

The issue is closed and yet not solved. Sorry but this is a letdown guys...

**Comment by Betterbird** [comment]
Please read https://www.betterbird.eu/support/. BB is a TB downstream project (fork) and we cannot possibly fix all 14.000 upstream issues.

As reported, the issues is not reproducible. However, we've given you useful details how to investigate it further. If reproducible, it **must** be reported as upstream issue first.

Users like yourself, who refuse to cooperate or support the project via a donation, are a **total** let-down indeed, especially if they feel entitled to get a fix. Note that we fixed your ticket #36 within days since it was reproducible and reported upstream.

**Comment by pglpm** [comment]
It's important to support projects and I indeed do that with all software I use (except Emacs). But my personal policy is first to use the software for a couple of months to see if it is really what I'm looking for. In this case I was looking for an email client to use _instead of_ Thunderbird. I understand now that in order to use Betterbird I do still have to worry about Thunderbird, even if for just reporting issues. So Betterbird is simply not what I was looking for. My bad.

I will of course uninstall it, it isn't nice to use something without at least contributing financially to it.

**Comment by Betterbird** [comment]
Your call.

Say good-bye to handling NBSP in your e-mails then. If you buy a tuned VW or BMW, it's still a VW or BMW under the hood and the tuning company won't fix all the issues of the product. You can use BB very well instead of TB with excellent responsive support as long as you adhere to the rules we have set. You haven't even given a reproducible case or investigated your issue with the hints we gave.

**Comment by dumblob** [comment]
@Betterbird could you set up a template for GitHub issues along these lines:

>### Mandatory checklist:
>
>- [ ] The issue already exists in the [upstream Thunderbird Bugzilla](https://bugzilla.mozilla.org/describecomponents.cgi?product=thunderbird ) - if not, please don't expect full support from our side.
>- [ ] I read https://www.betterbird.eu/support and acknowledge that without [financial donation or other material support](https://www.betterbird.eu/donations ) this issue might be closed as *non-actionable for the time being* **without further description**. This helps us a lot to prioritize our work (we're not just a group of volunteers). Thank you for understanding.
>
>### Steps to reproduce - incl. the unexpected or wrong outcome (detailed!):
>
>1. click on BB icon in a desktop menu (Gnome 3.1)
>2. wait for 35 seconds
>3. small empty window appears in the middle of the screen (it has title "BB could not be opened" and the content of the windows is clear white)
>
>### What is the expected or envisioned behavior (detailed!)?
>
>BB opens and works as described in documentation.
>
>### Any other related comments, links, ideas etc.
>
>1. related feature request in upstream: https://bugzilla.mozilla.org/describecomponents.cgi?product=thunderbird
>2. I love BB, thanks guys!

Note I've carefully read everything on www.betterbird.eu (indeed, I'm that crazy because I couldn't believe someone "just ") just a few days ago when I discovered BB for the first time and have to say nothing from the template above was clearly conveyed.

Thoughts?

**Comment by Betterbird** [comment]
Well, OK, we'll put something about support on the GitHub page. That said, what is unclear here (OK, we've just modified it a bit):
https://www.betterbird.eu/support

---

So what do to? In general, we're here to help, but as you will understand, our small team cannot fix all 10.000+ bugs given that some of them are two decades old and the Thunderbird team which has many more resources than we do haven't addressed those bugs.

As a first step, we suggest to install the current ESR 91 version of Thunderbird in parallel and see whether the issue exists there. If so, it's a good idea to check whether a bug has been reported already at [Bugzilla](https://bugzilla.mozilla.org/). If the issue exists in Thunderbird, regardless of whether it has been reported at or not, you can [contact us](javascript:linkTo_UnCryptMailto('nbjmup;tvqqpsuAcfuufscjse/fv');) and we can check how hard it would be to fix the issue. We may also advise you to file a bug in [Bugzilla](https://bugzilla.mozilla.org/), so we can offer our fix to upstream Thunderbird. We may chose it file it ourselves. Generally the idea of Betterbird is that our team tries to provide users with a better experience and that involves fixing long-standing and annoying bugs that somehow the Thunderbird team is not addressing.

If the issue only exists in Betterbird, please [contact us](javascript:linkTo_UnCryptMailto('nbjmup;tvqqpsuAcfuufscjse/fv');) immediately and we will endeavour to get a fix to you as soon as possible. Or you can file an issue at our [Github repository](https://github.com/Betterbird).

In any case, common sense bug reporting rules apply, so something that's not reproducible won't be fixed and the user needs to cooperate during the debugging phase by following our suggestions and providing feedback or debugging information.

---

To us the statements are pretty clear:
1. TB has 10.000+ bugs with won't be fixed as part of BB
2. First step: Check whether the bug exists in TB. If so, check whether it has been reported.
3. If the bug is also in TB, let us know that it's important to you, we'll check whether we deem it "must have" enough to fix it (like #36, still considering #35).
4. If the bug is only in BB, let us know, we'll endeavour to fix it asap, usually within days.
5. Common sense bug reporting rules apply.

**Comment by dumblob** [comment]
I for myself live on GitHub (i.e. I discover projects through GitHub links & search) and only rarely read project pages because I'm interested in the code.

And surely I'm not the only one as GitHub **is for code** and collaboration, so why to read any (more often than not) unmaintained, half-nonfunctional, marketing-like pages if it makes one usually frustrated even before one dives into the issue tracker or code.

Therefore if there is no clear articulation of the "rules" and "situation" **directly** when opening new issues on GitHub, then I can't really be the one to be blamed (nor the repo maintainers can be those to be blamed). It's status quo. But status quo is not really what a thriving project wants from my experience.

Therefore my suggestion to install a clear issue template **on GitHub** as outlined above :wink:.
